recipes/imagl/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ImaglTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
I think the failure is due to the missing `cmake_find_package` generator in the `test_v1_package`.

```suggestion
    generators = "cmake", "cmake_find_package_multi"
```